documentation/src/main/asciidoc/topics/proc_generating_protostream_sci.adoc

```diff
@@ -39,7 +35,15 @@ include::code_examples/Author.java[]
 include::code_examples/Book.java[]
 ----
 +
-. Define an interface that extends `SerializationContextInitializer` and is annotated with `@AutoProtoSchemaBuilder`.
+. If necessary, create an adaptor class for any external, third-party Java object classes and add the `@ProtoAdaptor` annotation.
++
+[source,java]
+----
+include::code_examples/ProtoAdaptor.java[]
+----
```

Comment:
I missed this part :upside_down_face:

Reply:
no worries. doc PRs can be difficult to follow that way. I'll generate some HTML and ping you the link so you can see the full rendered output.